src/Command/CleanUpCommand.php

```diff
@@ -55,10 +34,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if ($storage->cleanup(
             $input->getOption('now') ? new \DateTime($input->getOption('now')) : null
```

Comment:
You should have already provided the correct value type as fallback. This caused an additional null-check in your mapper code.